sweep/tx_input_set.go

```diff
@@ -24,6 +24,10 @@ const (
 	// constraintsWallet is for wallet inputs that are only added to bring up the tx
 	// output value.
 	constraintsWallet
+
+	// constraintsForce is for inputs that should be swept even with a negative
+	// yield at the set fee rate.
+	constraintsForce
```

Comment:
maybe not a use for it right now, but could be interesting to make this a bit field at some point st multiple constraints could be combined.

Reply:
maybe, but maybe wagni...